### PR TITLE
Testing pull request

### PR DESCRIPTION
**Pull request by JonathanKross**
fixing a bug!

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1 +1,5 @@
 print('Hello Claire')
+
+print('This is some fun practice')
```
Comment (benmccormick) [inline hello.py]: No way this is serious!!

Comment (JonathanKross) [inline hello.py]: but please!

**Comment by benmccormick** [comment]
Please fix the serious comment, or I won't merge it.

File: hello.py [hunk 2]
```diff
@@ -1 +1,5 @@
 print('Hello Claire')
+
+print('This is some super serious fun practice')
```
Comment (benmccormick) [inline hello.py:3]: Much better!

**Pull request merged**